project/Dependencies.scala

```diff
@@ -2,11 +2,22 @@ import sbt._
 
 object Dependencies {
 
-  // Generic
-  lazy val jodaTime = "joda-time" % "joda-time" % "2.7"
+  val playWsStandaloneVersion = "2.0.0-M1"
```

Comment:
I feel like this is too bleeding edge. We should stick to stable releases here. Isn't play 2.6 already compatible with scala 2.12?